keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifactSupplier.kt

```diff
@@ -57,25 +55,12 @@ class DebianArtifactSupplier(
         }
     }
 
-  override fun getArtifactByVersion(artifact: DeliveryArtifact, version: String): PublishedArtifact? =
+  override fun getArtifactByVersion(artifact: DeliveryArtifact, version: String): ArtifactVersion? =
     runWithIoContext {
       artifactService.getArtifact(artifact.name, version.removePrefix("${artifact.name}-"), DEBIAN)
     }
 
-  override fun getFullVersionString(artifact: PublishedArtifact): String =
```

Comment:
These methods are now getters in `ArtifactVersion` and no longer need to be replicated in different `ArtifactSupplier` implementations.